driver-reactive-streams/src/test/functional/com/mongodb/reactivestreams/client/syncadapter/SyncMongoClient.java

```diff
@@ -88,6 +89,18 @@ public static void enableSleepAfterSessionClose(final long sleepMS) {
         sleepAfterSessionCloseMS = sleepMS;
     }
 
+    public static void enableWaitForBatchCursorCreation() {
```

Comment:
@vbabanin can you suggest some Javadoc for this that describes its intent?  Or perhaps suggest a different name altogether for the methods?

Reply:
Sure, we could use this Javadoc to describe the intent of the method:

```
    /**
     * Enables behavior for waiting until a reactive {@link BatchCursor} is created.
     * <p>
     * When enabled, {@link SyncMongoCursor} allows intercepting the result of the cursor creation process.
     * If the creation fails, the resulting exception will be propagated; if successful, the 
     * process will proceed to issue `getMore` commands.
     * <p>
     *NOTE:  If tests are using several cursors concurrently, multiple threads might interfere with 
     * each other due to {@link Hooks#onEachOperator(Function)} functioning globally for any operator.
     */
```